.build/sh/change-submodule.sh

```diff
@@ -23,6 +23,11 @@ set -o nounset
 _usage() {
   cat <<EOF
 Usage: $(basename $0) [submodule path] [git repo URL] [git branch]
+
+Examples:
+
+Switch to branch immutable-state
```

Comment:
nitty nit: Maybe quotes around `immutable-state` for clarity...not super important